content/docs/2.1/scalers/mongo.md

```diff
@@ -0,0 +1,80 @@
+# mongo with ScaledJob
+
+## Trigger Specification
+This specification describes the mongo trigger that scales based on result of Mongo query.
+
+The trigger always requires the following information:
+- dbName  -Name of the database
+- collection -Name of the collection
+- query -a Mongo query that should return single numeric value
+- queryValue - a threshold that is used as targetAverageValue in HPA.
+
+To provide information about how to connect to Mongo you can provide
+- connectionStringFromEnv -Mongo connection string that should point to environment variable with valid value
+
+Or provide more detailed information:
+- host - The host of the Mongo server
+- port - The port of the Mongo server
+- username - Username to authenticate with to Mongo database
+- passwordFromEnv - Password for the given user
+## Authentication Parameters 
```

Comment:
```suggestion

## Authentication Parameters

```

Reply:
I think it should be ok now. @tomkerkhove ,pls review it again.